geode-core/src/main/java/org/apache/geode/internal/net/SocketCreator.java

```diff
@@ -1079,13 +1136,13 @@ private void configureClientSSLSocket(Socket socket, int timeout) throws IOExcep
         }
       } catch (SSLHandshakeException ex) {
         logger
-            .fatal(String.format("SSL Error in connecting to peer %s[%s].",
+            .fatal(String.format("Problem forming SSL connection to %s[%s].",
```

Comment:
We have a lot of these lingering around.  I prefer String.format() because it doesn't allow you to accidentally have the wrong number of arguments.  I had to fix a lot of those when I got rid of LocalizedStrings.  I'll change it to use varargs though.